be/src/olap/snapshot_manager.cpp

```diff
@@ -546,4 +576,34 @@ OLAPStatus SnapshotManager::_create_snapshot_files(
     return res;
 }
 
+OLAPStatus SnapshotManager::_convert_beta_rowsets_to_alpha(const TabletMetaSharedPtr& new_tablet_meta,
+                const vector<RowsetMetaSharedPtr>& beta_rowsets, const std::string& dst_path,
+                std::vector<RowsetMetaSharedPtr>* new_rowsets, bool* modified) {
+    OLAPStatus res = OLAP_SUCCESS;
+    RowsetConverter rowset_converter(new_tablet_meta);
+    for (auto& rowset_meta : beta_rowsets) {
```

Comment:
```suggestion
    for (auto& rowset_meta : rowset_metas) {
```
Not all rowsets are beta rowset, better to name it rowset_metas